scenes/pianist.fountain

```diff
@@ -0,0 +1,73 @@
+[Terry](character) runs off to play. [Howard](character) watches for a
```

Comment:
`(character)` is necessary only if the character is silent in a scene. IF the character has speaking lines, the software notices that without special markup.